versions.sh

```diff
@@ -17,8 +17,9 @@ declare -A alpineVersions=(
 	#[5.0]='3.16'
 )
 # see https://www.redmine.org/projects/redmine/wiki/redmineinstall
-defaultRubyVersion='3.1'
+defaultRubyVersion='3.2'
 declare -A rubyVersions=(
+  [5.0]='3.1'
```

Comment:
Minor whitespace nit:

```suggestion
	[5.0]='3.1'
```

(Should be tabs to match the rest of the file.)